Apps/phradar/radar.json

```diff
@@ -440,41 +441,29 @@
                 {
                     "data_path": "action_result.data.*.incident_id",
                     "data_type": "numeric",
-                    "column_name": "ID",
-                    "column_order": 0,
                     "contains": [
                         "radar incident id"
                     ]
                 },
                 {
                     "data_path": "action_result.data.*.name",
-                    "data_type": "string",
-                    "column_name": "Name",
-                    "column_order": 1
+                    "data_type": "string"
                 },
                 {
                     "data_path": "action_result.data.*.description",
-                    "data_type": "string",
-                    "column_name": "Description",
-                    "column_order": 2
+                    "data_type": "string"
                 },
                 {
                     "data_path": "action_result.data.*.discovered",
-                    "data_type": "string",
-                    "column_name": "Discovered At",
```

Comment:
As I understand it, the column names and order are being removed because the app is not using a default table template?

Reply:
Yes, that's correct.